protos/hardware/gen.go

```diff
@@ -0,0 +1,3 @@
+package hardware
+
+//go:generate moq -out mock.go . HardwareServiceClient HardwareService_AllClient
```

Comment:
boots uses https://github.com/golang/mock/gomock we should probably pick one for both projects.